test/climber/README.md

```diff
@@ -0,0 +1,1034 @@
+```solidity
+bytes32 id = getOperationId(targets, values, dataElements, salt);
+...
+operations[id].readyAtTimestamp = uint64(block.timestamp) + delay;
+operations[id].known = true;
+```
+Then, when we want to execute this `action`, we have to pass exactly the same parameters to `execute()`, **byte by byte** (this is important), as in `schedule()`, because here the hash will be taken again, the `id` will be calculated, and based on this it will be clear whether this action was registered at all:
+```solidity
+function getOperationState(bytes32 id) public view returns (OperationState state) {
+    Operation memory op = operations[id];
+
+    if (op.known) {
+        if (op.executed) {
+            state = OperationState.Executed;
+        } else if (block.timestamp < op.readyAtTimestamp) {
+            state = OperationState.Scheduled;
+        } else {
+            state = OperationState.ReadyForExecution;
+        }
+    } else {
+        state = OperationState.Unknown;
+    }
+}
+```
+```solidity
+bytes32 id = getOperationId(targets, values, dataElements, salt);
+...
+if (getOperationState(id) != OperationState.ReadyForExecution) {
+    revert NotReadyForExecution(id);
+}
+```
+Even if we find a way to somehow register the `operation` (reminding: `attacker` is neither an `admin` nor a `proposer`) - we still need to fulfill two conditions to properly prepare the test for symbolic execution:
+1. Bypass symbolic offset error in `execute()`. The "antidote" to it is to use the `CreateCalldata()` cheatcode to replace `dataElements[]`. 
+2. `dataElements[]` bytes in the `schedule()` should be replaced not via `CreateCalldata()`, but via `CreateBytes()` cheatcode. The thing is, this seems more correct: we can really pass literally any bytes here. This is especially true in setups with upgradable proxy (nothing prevents us from creating an `operation` for proxy, which is not supported by the current **implementation** yet, but maybe someday it will be). And using `CreateCalldata()` imposes restrictions on what these bytes can be. Although I admit that this take can be debatable.
+
+Such a mismatch in different cheatcodes in `schedule()` and `execute()` can badly affect the logic of finding the `id`. The fact is that the bytes generated by `CreateCalldata()` can be of completely different lengths, while `CreateBytes()` requires us to clearly specify the number of generated symbolic bytes ("extra" bytes will be replaced with `0s` as padding).
+
+Now let's remember that we need to preserve the identity of the parameters of both functions to correctly schedule and execute the `operation`. Thus, we will not be able to simply prove that the `operation` we passed to `execute()` was scheduled. We are faced with an unusual problem that needs to be solved.
+
+Here are some ideas on how we can solve this:
+1. Somehow modify symbolic bytes from `CreateCalldata()` so that they are also of static size and output `0s` at the end as padding.
```

Comment:
this seems like a pretty good idea, have you tried it? Seems like this would work:

```
// will be case split into multiple sizes
bytes memory data = svm.createCalldata(...);

// make it an explicit fixed size by padding with zeros:
bytes memory padding = 1024 - data.length;
bytes memory padded = bytes.concat(data, padding);
```

and there you go, you should have symbolic calldata of size 1024 padded with the appropriate number of 0s